### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by JantuDeb**
# Alert
- Added alerts for actions.
- refactored some code.
- fixed the playlist issue.

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
code looks clean. liked that you are using different types of toast for different purposes

File: src/context/history/HistoryContext.jsx [hunk 1]
```diff
@@ -28,7 +29,8 @@ const HistoryVideoProvider = ({ children }) => {
         });
       }
     } catch (error) {
-      console.log(error);
+      if (error.response)
+        toast.error(error.response?.data?.message, { autoClose: 2000 });
```
Comment (sushil-mod) [inline src/context/history/HistoryContext.jsx:33]: i Liked  how you used short code to display toast

**Review by sushil-mod — APPROVED** [review]
Toast functionality working fine . but in preview on starting bunch of toast comes( check it once) . rest code is good

**Comment by JantuDeb** [comment]
> Toast functionality working fine . but in preview on starting bunch of toast comes( check it once) . rest code is good

Thanks , will fix it

**Pull request merged**